Amethyst/default.amethyst

```diff
@@ -166,6 +166,8 @@
         "mod": "mod2",
         "key": "0"
     },
+    "throw-space-11": [],
```

Comment:
You should actually just remove these. They will still default to not being set.

Reply:
kk back to you. :)